quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -342,7 +383,37 @@ function ziti_expressConfiguration {
     echo " "
     if [[ "${REPLY}" == [Yy]* ]]; then
       echo -e "$(GREEN "Clearing existing Ziti variables and continuing with express install")"
+
```

Comment:
This was necessary in order to handle errors "nicely". If you already have ziti version A installed and you try to install version B, the database version mismatches and throws an error when trying to start up the controller.

In trying to keep with the "quickstart" mindset I felt that (even though this is a lot of extra logic) it would make the end user experience easier by not requiring them to read the log and guess what was going wrong. It also makes the end user experience better in the event they want to reinstall but keep the specific version (which would be wiped out by the `unsetZitiEnv` function.

With this being said, feedback on alternative solutions is welcomed.